course_inventory/inventory.py

```diff
@@ -182,6 +182,31 @@ def gather_course_data_from_api(account_id: int, term_ids: Sequence[int]) -> pd.
     return course_df
 
 
+def gather_account_data_from_api(account_ids: Sequence[int]) -> pd.DataFrame:
+    logger.info('** gather_account_data_from_api')
+    url_ending_with_scope = f'{API_SCOPE_PREFIX}/accounts/'
+
+    account_dicts = []
+    for account_id in account_ids:
+        logger.info(f'Account number {account_id}')
+        account_url_ending = url_ending_with_scope + str(account_id)
+        response = make_request_using_api_utils(account_url_ending)
+        account_data = json.loads(response.text)
```

Comment:
The JSON load can raise an exception. should handle it

Reply:
This is already being handled in `make_request_using_api_utils`: https://github.com/tl-its-umich-edu/course-inventory/blob/aa9bfa7e1307ed564df16a8eb5b3b487fbc8ffae/course_inventory/inventory.py#L49-L74

Reply:
@pushyamig, does this not resolve your concern? Or maybe you just didn't see the comment?

Reply:
Do we want it to continue if all three attempts fail? I'm logging an error and return a response for consistency, but I was figuring that the code would break at that point, which is probably what we want. If we want to use `sys.exit(1)` inside the function, I'm okay with that. We haven't encountered this case yet in my memory.

Reply:
Depending on your answer, I think this might be best addressed in a separate issue. I'm not modifying this function at all in this PR.

Reply:
I think I had this in my original implementation, but when using the paging functionality of `umich_api`, you need to pass a response object to `API_UTIL.get_next_page`. See this code from `gather_course_data_from_api`: https://github.com/tl-its-umich-edu/course-inventory/blob/d4984bc91c0253955ec3b8b146e8882d203f5229/course_inventory/inventory.py#L149-L166

Reply:
OK.  Makes sense.

Reply:
@pushyamig, I've opened issue #161 so we can continue discussing what you raised here.